src/EventTile.tsx

```diff
@@ -74,32 +74,40 @@ export const EventTile: React.FC<EventTileProp> = ({ item, continuation }) => {
             body = "Redacted";
         } else if (MsgLikeKind.UnableToDecrypt.instanceOf(message.kind)) {
             body = "UTD";
-        } else if (
-            MsgLikeKind.Message.instanceOf(message.kind) &&
-            MessageType.Text.instanceOf(message.kind.inner.content.msgType)
-        ) {
+        } else if (MsgLikeKind.Message.instanceOf(message.kind)) {
             if (
-                message.kind.inner.content.msgType.inner.content.formatted
-                    ?.body &&
-                message.kind.inner.content.msgType.inner.content.formatted
-                    ?.format?.tag === MessageFormat_Tags.Html
+                MessageType.Image.instanceOf(message.kind.inner.content.msgType)
             ) {
-                const html = sanitizeHtml(
+                const mxc =
+                    message.kind.inner.content.msgType.inner.content.source.url();
+                body = <img src={mxcToUrl(mxc, 200)} height={250} />;
```

Comment:
this looks weird - you're asking for an image smaller than 200px, and then forcing it to be 250px high?

Reply:
Good point, originally had it rendering at a measly 100px @2xhidpi